pymemcache/client/retrying.py

```diff
@@ -0,0 +1,183 @@
+""" Module containing the RetryingClient wrapper class. """
+
+from collections import Iterable
```

Comment:
The library currently supports Python 2.7 which doesn't have `collections.abc`, but I don't see any reason why it can't be done with a `try .. except`. 👍

Reply:
Oh yes good catch. My bad.
I was thinking that the py2 support was already dropped, but indeed that's not the case yet

https://github.com/pinterest/pymemcache/pull/321

Reply:
Import became unnecessary after the check simplification below, so just removed it. Nicely spotted.